hic_qc.py

```diff
@@ -314,10 +317,21 @@ def extract_header_info(self, header, xx=50):
                 elif token_proc.endswith('_r1.fastq') or token_proc.endswith('_r1.fq') \
                     or token_proc.endswith('_r1.fastq.gz') or token_proc.endswith('_r1.fq.gz'):
                     self.fwd_hic_reads = os.path.basename(token)
+                    full_fwd_reads = token.strip()
                 elif token_proc.endswith('_r2.fastq') or token_proc.endswith('_r2.fq') \
                     or token_proc.endswith('_r2.fastq.gz') or token_proc.endswith('_r2.fq.gz'):
                     self.rev_hic_reads = os.path.basename(token)
-            if self.fwd_hic_reads == self.rev_hic_reads:
+                    full_rev_reads = token.strip()
+            files_matched = False
+            if os.path.exists(full_fwd_reads) and os.path.exists(full_rev_reads):
```

Comment:
Would this handle the case where a client accidentally uses the same file for forward and reverse reads? It looks to me like it would fail to find the fwd or rev reads, and then not detect that they matched

Reply:
In this case, `self.rev_hic_reads` (or `fwd_hic_reads` if they gave two R2's) would stay as "reverse Hi-C reads not found" and then be surfaced in the report that way, because the `bwa mem` command in the BAM header would have the same file name listed twice. Tested this with a BAM with this in the header:

`@PG	ID:bwa	PN:bwa	VN:0.7.15-r1144-dirty	CL:/home/ec2-user/PGTools/bwa/bwa mem -5 -t 15 /home/ec2-user/PGScratch/abc_qc_control2/shotgunized_abc_mix.fasta /home/ec2-user/PGScratch/abc_qc_control2/abc_qc_control2_S3HiC_R1.fastq.gz /home/ec2-user/PGScratch/abc_qc_control2/abc_qc_control2_S3HiC_R1.fastq.gz`